recipe/activate-clang_win-64.bat

```diff
@@ -0,0 +1,101 @@
+@echo on
```

Comment:
Can you depend on `vs2017_win-64` so that we don't need to duplicate those stuff here?

Reply:
This doesn't work for me as the `vs2017_win-64` activation gets called after `activate-clang_win-64.bat`. I'm not sure how `conda` is resolving the activate ordering.

Reply:
It's dependency order and if there's a cycle alphabetical.

Reply:
At the moment it is in alphabetical order, for dependency order there is still an open issue: https://github.com/conda/conda/issues/6338